packages/plugin-session-replay/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@amplitude/plugin-session-replay",
+  "version": "1.0.0-beta.0",
+  "description": "",
+  "author": "Amplitude Inc",
+  "homepage": "https://github.com/amplitude/Amplitude-TypeScript",
+  "license": "MIT",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "types": "lib/esm/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
```

maybe we can also set the dist tag to beta until we ga?

```suggestion
    "access": "public",
    "tag": "beta"
```